keras_nlp/models/bart/bart_tokenizer_test.py

```diff
@@ -74,7 +74,16 @@ def test_errors_missing_special_tokens(self):
         with self.assertRaises(ValueError):
             BartTokenizer(vocabulary=["a", "b", "c"], merges=[])
 
+    def test_serialization(self):
+        config = keras.saving.serialize_keras_object(self.tokenizer)
+        new_tokenizer = keras.saving.deserialize_keras_object(config)
+        self.assertEqual(
+            new_tokenizer.get_config(),
+            self.tokenizer.get_config(),
+        )
+
     @pytest.mark.large  # Saving is slow, so mark these large.
+    @pytest.mark.tf_only
```

Comment:
This is because a string model inputs are only supported in tf. So a model that includes tokenization inside call really only makes sense in tf.

I am also open to just deleting these tests. I think saving for backbone models and tasks are important, for tokenizers saving directly like this less so.

Reply:
We could potentially rewrite the test with `preprocessor=False` to make it generic

Reply:
Oh this is actually a test for just the tokenizer. So `preprocessor=False` would not apply here.